packages/client-core/src/admin/common/Location/LocationDrawer.tsx

```diff
@@ -202,28 +261,13 @@ const LocationDrawer = ({ open, mode, selectedLocation, selectedScene, onClose }
       isFeatured: false //state.isFeatured.value
     }
 
-    state.formErrors.merge({
-      name: state.name.value ? '' : t('admin:components.location.nameCantEmpty'),
-      maxUsers: state.maxUsers.value ? '' : t('admin:components.location.maxUserCantEmpty'),
-      scene: state.scene.value ? '' : t('admin:components.location.sceneCantEmpty'),
-      type: state.type.value ? '' : t('admin:components.location.typeCantEmpty')
-    })
+    const onError = (error) => NotificationService.dispatchNotify(error.message, { variant: 'error' })
 
-    if (validateForm(state.value, state.formErrors.value)) {
```

Comment:
Don't delete the validateForm function, this removes notifications about incorrectly entered fields